recipes/nrtk-explorer/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "nrtk-explorer" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nrtk_explorer-{{ version }}.tar.gz
+  sha256: 05a0c81476df66979b4a6fc23814a6b4e8589573115b910aabf667b8fac092ac
+
+build:
```

Comment:
Are you sure this cannot be noarch?

When I generate this with Grayskull I get:


```yaml
{% set name = "nrtk-explorer" %}
{% set version = "0.1.2" %}

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nrtk_explorer-{{ version }}.tar.gz
  sha256: 05a0c81476df66979b4a6fc23814a6b4e8589573115b910aabf667b8fac092ac

build:
  entry_points:
    - nrtk-explorer = nrtk_explorer.app:main
    - nrtk-explorer-embeddings = nrtk_explorer.app.embeddings:embeddings
    - nrtk-explorer-tranforms = nrtk_explorer.app.transforms:transforms
    - nrtk-explorer-filtering = nrtk_explorer.app.filtering:filtering
  noarch: python
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
  number: 0

requirements:
  host:
    - python >=3.7
    - hatchling
    - pip
  run:
    - python >=3.7
    - nrtk
    - numpy
    - pillow
    - scikit-learn ==1.4.1.post1
    - smqtk-classifier ==0.19.0
    - accelerate
    - smqtk-core ==0.19.0
    - smqtk-dataprovider ==0.18.0
    - smqtk-descriptors ==0.19.0
    - smqtk-detection ==0.20.1
    - smqtk-image-io ==0.17.1
    - tabulate
    - transformers
    - timm
    - pytorch
    - torchvision
    - trame
    - trame-client >=2.15.0
    - trame-quasar
    - trame-server >=2.15.0
    - ubelt ==1.3.4
    - umap-learn

test:
  imports:
    - nrtk_explorer
  commands:
    - pip check
    - nrtk-explorer --help
    - nrtk-explorer-embeddings --help
    - nrtk-explorer-tranforms --help
    - nrtk-explorer-filtering --help
  requires:
    - pip

about:
  summary: Model Visualizer
  license: Apache-2.0
  license_file: LICENSE

extra:
  recipe-maintainers:
```

Reply:
The reason for removing the noarch is that I need to skip the windows build since Pytorch/torchvision (python pkgs that we depend on) does have provide packages for windows in conda-forge (they have their own external repo for windows).

Reply:
@BastianZim

Reply:
@BastianZim I was able to keep the package noarch by using virtual packages `__unix` and removing the windows ci pipeline. Ready for review

Reply:
If a dependency is missing, it is fine if that check fails. That way, if the dependency gets added later, the package will become installable there as well.

Reply:
I see, Thanks for the review. Find the requested changes in the latest push

Reply:
@BastianZim ready to merge, note that the failing windows build is due to the changes that you requested.